src/templates/blog-post.tsx

```diff
@@ -69,7 +69,7 @@ const BlogPostPage: React.FC<IBlogPostPageProps> = ({ data }) => {
         image={
           picture &&
           (isProduction
-            ? `/blog/${picture.childImageSharp.fields.sourcePath}`
+            ? `/blog/images/${picture.childImageSharp.fields.sourcePath}`
```

Comment:
The `sourcePath` at some point changed, breaking how we generated our social image paths. Maybe when #3396 was merged, but I'd need to debug further to be sure.

Reply:
Pretty sure it was 3396, I remember having to change something to do with sourcepath in order to fix a build-breaking issue